cli/init/eth-contracts.js

```diff
@@ -1,6 +1,8 @@
 const BN = require('bn.js')
 const fs = require('fs')
-const { Web3, normalizeEthKey } = require('rainbow-bridge-utils')
+const { Web3, normalizeEthKey, sleep } = require('rainbow-bridge-utils')
+
+const RETRY_SEND_TX = 15
```

Comment:
Where does this constant come from?

Reply:
It is just a magic number like [this](https://github.com/aurora-is-near/rainbow-bridge/blob/master/utils/robust.js#L8) and [this](https://github.com/aurora-is-near/rainbow-bridge/blob/fd7019ff001968105e8bb22627f5a5097ba345a8/utils/helpers.js#L5) to get around of nonce error.